aws/resource_aws_codebuild_project.go

```diff
@@ -59,6 +59,25 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 				},
 				Set: resourceAwsCodeBuildProjectArtifactsHash,
 			},
+			"cache": {
+				Type:     schema.TypeSet,
```

Comment:
Since there won't ever be more than 1 `cache` stanza we should make it `TypeList` instead as there's no point in computing the hash for a set and dealing with ordering (as there's nothing to order). Also it will be ~easier~ possible to reference nested fields like `${aws_codebuild_project.cache.0.type}` etc.

Reply:
Ok, I've changed it into a List now, seems to work fine. Thanks for the tip!